### PR TITLE
First draft.

### PR DESCRIPTION
**Pull request by calleluks**
This is a first draft of the presentation for my introductory talk at the Rails intro course next week.

It is loosely based on [Gunnars presentation](http://slides.com/gudj/rails-intro-short/) that we've used previous times. I have removed information that wasn't used the last time and added some more instructions for attendees.

The project is based on `jekyll` and can be deployed as a static html site using GitHub pages. To see what this project looks like in your browser, install `jekyll` clone down the repo check out this branch and run

```
jekyll serve
```

in the project directory to start the server. You can now visit the address output by `jekyll serve` in your browser.

I'm would very much appreciate feedback of any kind. My goal is that this presentation can be used for all the coming intro courses and that I'll not be the only one using it.

### CODE REVIEW

**Comment by calleluks** [comment]
A note about the Events slide: I have not yet added a list of upcoming events since I know that @linduxed and @teoljungberg are working on updating the schedule right now.

**Comment by Koronen** [comment]
Over all, I think this looks good. However, considering this presentation will be displayed on a projector, maybe we could bump the font sizes up a bit? It might also be worth mentioning somewhere that you need Jekyll 2.x for SCSS support.

**Comment by calleluks** [comment]
Thanks @Koronen. I will manually adjust the font size to match the size and aspect ratio of the projector using the built in browser functionality during the presentation. Partly because I don't know the size and aspect ratio of the projector and partly because the first slide requires a smaller font size than the other slides.

Yes, I think adding a README that explains these things is a good idea. I'll fix that tomorrow.

**Comment by linduxed** [comment]
> I have not yet added a list of upcoming events since I know that @linduxed and @teoljungberg are working on updating the schedule right now.

I have replaced the Presentation event with the Project night, so this should no longer be a problem.

File: index.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+**Password**: `unfors33N`
+
+Step 3: Open this presentation in your browser
+----------------------------------------------
+
+This presentation contains links that you will need to access the various
+resources used in the course curriculum. You can access this presentation at
+the URL below:
+
+[http://intro.stockholmruby.com](http://intro.stockholmruby.com)
```
Comment (henrik) [inline index.md]: No practical difference but good to know: Markdown has a shortcut for a linkified URL: `<http://intro.stockholmruby.com>`

Some Markdown flavors will auto-link URLs even without any markup, I believe.

File: meta-goals.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+layout: default
+prev: getting-it-online
+next: schedule
+---
+
+When you have completed today's goal of getting the basic application online you
+should understand:
+
+* Basic Ruby syntax.
+* How to try your Ruby code (`irb(1)`).
```
Comment (henrik) [inline meta-goals.md]: Maybe just `irb` without the `(1)` is less confusing to some people?

File: what-are-we-building.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+title: What are we building?
+layout: default
+justification: center
+prev: what-is-a-web-application
+next: getting-it-online
+---
+
+An application for managing topics.
```
Comment (henrik) [inline what-are-we-building.md]: "Managing topics" is a bit generic. Maybe "Listing topics to vote on"? At least if the voting part comes into play.

File: index.md (hunk 1 above)
Comment (calleluks) [inline index.md]: Thanks, I did not know that! Updated to use the shorthand in https://github.com/sthlmrb/rails-intro-course-presentation/commit/f325eb2f9eb57eacd1e1c06b060c1f07fffc2c14.

File: meta-goals.md (hunk 2 above)
Comment (calleluks) [inline meta-goals.md]: Yes, that might be the case. Updated in https://github.com/sthlmrb/rails-intro-course-presentation/commit/95bfa3794f90905248c5161f5b48abff6a80e40f.

File: what-are-we-building.md (hunk 3 above)
Comment (calleluks) [inline what-are-we-building.md]: Good point. Updated in https://github.com/sthlmrb/rails-intro-course-presentation/commit/95bfa3794f90905248c5161f5b48abff6a80e40f.

**Comment by calleluks** [comment]
@henrik Thanks for reviewing. I think this is ready for another pass.

@Koronen I have now added a readme. Is there anything you think I should add to it?

**Comment by Koronen** [comment]
LGTM! :+1:

**Comment by teoljungberg** [comment]
The only thing I can nag about is that we should have a `Rakefile` instead of the `Makefile`

**Comment by georgebrock** [comment]
> The only thing I can nag about is that we should have a Rakefile instead of the Makefile

Why? Nothing wrong with make.

**Comment by teoljungberg** [comment]
@georgebrock I just find it ironic that a ruby-group is using a non-ruby tool
`</nag>`

**Comment by georgebrock** [comment]
@teoljungberg In my opinion the idea that you should use tools written in Ruby with Ruby projects is harmful. There are plenty of _great_ tools that are written in C, or Python, or whatever, and it's sad that we stay in our language-specific bubbles.

**Comment by calleluks** [comment]
Thanks everyone for reviewing. I've added a list of upcoming events. Squashing and merging.

**Pull request merged**